src/domain/safe/entities/schemas/delegate.schema.ts

```diff
@@ -0,0 +1,13 @@
+import { JSONSchemaType } from 'ajv';
+import { Delegate } from '../delegate.entity';
+
+export const delegateSchema: JSONSchemaType<Delegate> = {
+  type: 'object',
+  properties: {
+    safe: { type: 'string' },
```

Comment:
`safe` should be an optional field

Reply:
Done [here](https://github.com/5afe/safe-client-gateway-nest/pull/123/commits/f796ea02dd39a5701bcd760dda7c8924c0dcdf4c)